test/CMakeLists.txt

```diff
@@ -15,14 +15,14 @@ target_include_directories(CatchMain SYSTEM PUBLIC
 
 # Link other tests against this library to avoid repeated compilation of Catch2.
 add_library(vikunjaTestLib dummy.cpp)
-target_include_directories(vikunjaTestLib SYSTEM PUBLIC
-        ${CMAKE_CURRENT_SOURCE_DIR}/../include)
-target_include_directories(vikunjaTestLib SYSTEM PUBLIC ${alpaka_INCLUDE_DIR})
-target_include_directories(vikunjaTestLib SYSTEM PUBLIC
-        ${CMAKE_CURRENT_SOURCE_DIR}/include)
-target_include_directories(vikunjaTestLib SYSTEM PUBLIC ${alpaka_INCLUDE_DIR}/../test/common/include/)
-target_link_libraries(vikunjaTestLib PUBLIC "alpaka" "CatchMain")
+target_include_directories(vikunjaTestLib
+  SYSTEM PUBLIC
+  ${CMAKE_CURRENT_SOURCE_DIR}/include)
+target_include_directories(vikunjaTestLib
+  SYSTEM PUBLIC
+  ${alpaka_INCLUDE_DIR}/../test/common/include/)
```

Comment:
Why are these `SYSTEM` includes?

Reply:
I believe, it has something to do, if you use `#include <header.h>` instead `#include "header.h"` in your project, but I can check, if it is necessary. Maybe it was necessary, because the vikunja headers was direct include via cmake variable, instead using a target.

Reply:
This looks shaky. What if I point to an installed alpaka on my system? There shouldn't be a `test` include folder there.

Reply:
Yes, I know. But you are pointing to alpaka's `test` directory, right? What if there is no `test` directory because I pointed to an alpaka installation?

Reply:
Lines 21-23 seem like some leftovers, the test compile without them.

Reply:
Removed `SYSTEM` and the lines 21-23.